docs/modules/ROOT/nav.adoc

```diff
@@ -10,4 +10,5 @@
 
 .Release Notes
 
-* xref:release-notes.adoc[5.0]
+* xref:release-notes-5.0.adoc[5.0]
+* xref:release-notes-5.1.adoc[5.1]
```

Comment:
I think it's better to replace 5.0 release notes with 5.1. The 5.0 release notes belong in the 5.0 version of the docs.

Reply:
Ah, I see. I will change it.